guides/release/routing/specifying-a-routes-model.md

```diff
@@ -224,3 +343,14 @@ export default Route.extend({
 ```
 
 And calling `modelFor` returned the result of the `model` hook.
+
+## Debugging models
+
+If you are having trouble getting a model's data to show up in the template, here are some tips:
+
+- return static, sample data as a test to see if the problem is really in the model hook, or elsewhere down the line
+- study JavaScript Promises in general, to make sure you are returning data from the Promise correctly
+- make sure your `model` hook has a `return`
+- check to see whether the data returned from a `model` hook is an object, array, or JavaScript Primitive. For example, if the results of `model` are an array, using `{{this.model}}` in the template won't work. You will need to iterate over the results with an `{{#each}}` helper. If the results of `model` are a single object, you need to access the individual attributes like `{{this.model.title}}` to see anything rendered
+- use your browser's development tools to examine the outgoing and incoming API responses and see if they match what your code expects
+- if you are using Ember Data, use the [Ember Inspector](../../ember-inspector/) browser plugin to see if the records made it into the store
```

Comment:
If you're going to mention the ember inspector, the "model" column in the "View tree" tab would be the "go-to" place for this. Only if I saw that the model was null/undefined/empty would I want to check out loaded e-d records

 
<img width="942" alt="Screen Shot 2019-03-30 at 11 48 17 PM" src="https://user-images.githubusercontent.com/558005/55285750-604d3700-5346-11e9-8e29-294e0a28c550.png">

Reply:
This is interesting, I always ignored that column as gibberish.